guides/v2.3/rest/tutorials/msi-order-processing/create-cart-add-products.md

```diff
@@ -135,21 +135,23 @@ In this call, we'll add 20 `sp1` items. This portion of the order can be fulfill
   "cartItem": {
     "sku": "sp1",
     "qty": 20,
-    "quote_id": "1"
+    "quote_id": "10"
   }
 }
 ```
 
 **Response**
 
+Make a note of the `item_id`. It will be used in subsequent steps.
```

Comment:
Promises of the future? 
Maybe...
"Note the `item_id` for use in subsequent steps."